docs/content/en/docs/packages/overview.md

```diff
@@ -42,3 +49,38 @@ Please check out [package management]({{< relref "../packages/packages" >}}) for
 
 ### Curated packages artifacts
 There are three types of build artifacts for packages: the container images, the helm charts and the package bundle manifests. The container images, helm charts and bundle manifests for all of the packages will be built and stored in EKS Anywhere ECR repository. Each package may have multiple versions specified in the packages bundle. The bundle will reference the helm chart tag in the ECR repository. The helm chart will reference the container images for the package.
+
+
+### Installing packages on workload clusters
```

Comment:
Maybe this is way too verbose and just makes things worse? What do we think?

I feel like I could cut all the words and just have the image and example package spec and it might communicate how things work more succinctly/clearly.

Reply:
Yeah, I think there could be less words. It's not necessarily that it's too verbose, but personally when reading it i find myself trying to visualize things. Aside from a brief introduction to the package controller and it's responsibilities, I think how we can approach this is maybe we have images or terminal outputs and words should simply explain what's in the output/image.

Reply:
Also, should this information go under Package Management page? This page already links to in the first paragraph

Reply:
Did what you suggested and I think it's way better.